### ISSUE TITLE
Image Synthesis and 

### ISSUE THREAD

**Issue by masashi-hatano** [masashi-hatano/proj-mi-group17#1]
This issue is the first step to start our project.
We should decide who is gonna take care of which task in the beginning.
Of course, we can do together if some of us has difficulties.

TODO LIST:
- [ ] Preparation of images and texts:
We can get images from the existing datasets such as coco, ade20k, or cityscapes.
As for the text file, which contains a great number of words and numbers, we need to create on our own while taking into account the bias of number of letters.
- [ ] Generation of synthesized image:
By using an image and a text and mixing them up together, we can get a synthesized image.
For images, maybe we can crop them if the number of them is not enough.
When putting up together, we need to keep the location of bounding box of each character because it leads to efficient label generation.
- [ ] Creation of dataset:
We need to prepare a dataset for training ocr model; therefore, defining datamodule(or dataloader if we don't use pytorch lightning) and datasets is needed to be done.

**Comment by masashi-hatano** [comment]
Do you have any preferences for the tasks?

**Comment by masashi-hatano** [comment]
Do you have any preferences for the tasks?
